### PR TITLE
Trails and Pathways

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for the [Utah Trails and Pathways](https://gis.utah.gov/products/sgid/recreation/trails-pathways/) layer, with some of my comments and questions mixed in. This PR also includes a schema document for this layer's field definitions, let me know if there are corrections that need to be made or information that needs to be added!

### CODE REVIEW

File: metadata/recreation/trails_and_pathways.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+a341f781-d092-4079-b0fb-7d4da5eee5ea
+
+## Brief Summary
+
+Polyline dataset of established motorized and non-motorized trails in Utah.
+
+## Summary
+
+Utah Trails and Pathways contains off-road recreational trail features that are primarily used by hikers, pedestrians, and mountain bikers. UGRC developed this dataset using data from a variety of sources including Orbital View, the National Park Service, and the US Forest Service.
```
Comment (gregbunce) [inline metadata/recreation/trails_and_pathways.md:15]:
```suggestion
Utah Trails and Pathways contains off-road recreational trail features that are primarily used by hikers, pedestrians, and mountain bikers. UGRC developed this dataset using data from a variety of sources including Orbital View, the National Park Service, US Forest Service, as well as input from state agencies and metropolitan planning organizations.
```

File: metadata/recreation/trails_and_pathways.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Description
+
+### What is the dataset?
+
+This dataset depicts over 20,000 trail and pathway features used mostly by hikers/pedestrians and bikers. Some Off-Highway Vehicle (OHV) trails may be included. It does not include on-street bike infrastructure, sidewalks, or pedestrian bridges.
```
Comment (gregbunce) [inline metadata/recreation/trails_and_pathways.md:21]: it's worth noting that the on-street infrastructure can be found in the Utah Roads dataset.

File: metadata/recreation/trails_and_pathways.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+This dataset serves as a resource for recreational and city planning, hiking and mountain biking search engines, citizens, and local government authorities. It is suitable for cartographic and general analytic purposes.
+
+### What does the dataset represent?
+
+Each trail or pathway is represented as a polyline segment that includes attributes such as the name, allowed use information, and trail type. The primary use of the trail is specified by the CartoCode field.
+
+### How was the dataset created?
+
+UGRC created this dataset using map data purchased from Orbital View by the GOED Office of Outdoor Recreation. Since this original purchase, trails have been added from a variety of sources including the National Park Service, local county authorities, the US Forest Service, and the [Mountainland Association of Governments](https://mountainland.org/). This dataset is maintained by UGRC in collaboration with the [Wasatch Front Regional Council](https://wfrc.org/). Additional trails can be [submitted](https://gis.utah.gov/contact/) to UGRC for inclusion in this map layer.
```
Comment (gregbunce) [inline metadata/recreation/trails_and_pathways.md:33]: if we're calling out MAG, then we should call out UDOT and WFRC, too. else: we could say other state agencies and metropolitan planning organizations. either way.

File: metadata/recreation/trails_and_pathways.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+### How reliable and accurate is the dataset?
+
+Although it is a work in progress, this dataset represents a substantial starting point to integrate additional trail systems from other local, state, and federal agencies. The success of this effort will rely heavily on local government and land stewards to review and continue to improve upon this data.
+
+<!--- This is the original language of the metadata. Is this dataset still considered a "starting point"? Are we relying on local government and land stewards to maintain this dataset, or are we the primary driving force in updating and improving it? -->
```
Comment (gregbunce) [inline metadata/recreation/trails_and_pathways.md:39]: yeah, crazy enough, it still is a work in progress.

File: metadata/recreation/trails_and_pathways.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+## Credits
+
+### Data Source
+
+- UGRC
+- Local Government Authorities
+- US Forest Service
+- National Park Service
+- Orbital View
+
```
Comment (gregbunce) [inline metadata/recreation/trails_and_pathways.md:50]:
```suggestion
- State agencies
- Metropolitan Planning Organizations
```

File: metadata/recreation/trails_and_pathways.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+## Restrictions
+
+## License
+
+## Tags
+
+- Hiking
+- Mountain Biking
+- OrbitalView
```
Comment (gregbunce) [inline metadata/recreation/trails_and_pathways.md:63]:
```suggestion
```

**Review by gregbunce — CHANGES_REQUESTED** [review]
nice work. a few suggestions and comments. the schema flle looks good.

File: metadata/recreation/trails_and_pathways.md (hunk 4 above)
Comment (miriamseely) [reply metadata/recreation/trails_and_pathways.md:39]: Good to know! I'll keep it vague.

**Pull request merged**